authors/apps/articles/views.py

```diff
@@ -192,6 +194,13 @@ def delete(cls, request, article_id):
                 Article, id=article_id, delete_status=False)
             article.delete()
             return Response({"messege": "article deleted successfully"}, status=status.HTTP_200_OK)
+        if current_user.is_superuser:
+            try:
+                reported_article = ReportArticle.objects.get(article=article.slug, report_status="resolved")
+                article.delete()
```

Comment:
@llwasampijja, What do you think about doing a soft delete for a reported article so that the author can still view the article?